collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2157,6 +2160,11 @@ protected void onResume() {
             return;
         }
 
+        if (questionFontSize != Collect.getQuestionFontsize()) {
```

Comment:
It would be good to factor it out to a separate method, `onResume()` is big enough... our goal should be to have small methods with good names to read the code like a story. I know it might be difficult to find such an approach especially looking at `ForEntryActivity` class but that's a good practice.

Reply:
The name could be `adjustFontSize` for example.

Reply:
Done. Thanks :)